README.md

```diff
@@ -1,16 +1 @@
-# Group 4 - CMPE 352/451 2020 - "Buyo"
-
-![](https://github.com/bounswe/bounswe2020group4/blob/master/cmpe352/logoV1.png "Buyo")
-
-This is the repository of Group 4 for the course CMPE352/451. For details, visit our [wiki page](https://github.com/bounswe/bounswe2020group4/wiki).
-
-## Group Members
-| <img src="https://avatars.githubusercontent.com/burakcuhadar" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/eridincu" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/koraycetiin" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/eylulyalcinkaya" alt="alt text" width="150"> | 
-|---|---|---|---|
-|[Burak Çuhadar](https://github.com/bounswe/bounswe2020group4/wiki/Burak-%C3%87uhadar)|[Mehmet Erdinç Oğuz](https://github.com/bounswe/bounswe2020group4/wiki/Mehmet-Erdin%C3%A7-O%C4%9Fuz)|[Koray Çetin](https://github.com/bounswe/bounswe2020group4/wiki/Koray-Cetin)|[Eylül Yalçınkaya](https://github.com/bounswe/bounswe2020group4/wiki/Eyl%C3%BCl-Yal%C3%A7%C4%B1nkaya)|
-| <img src="https://avatars.githubusercontent.com/egirgin" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/berkeozdemir2016400246" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/katakor" alt="alt text" width="150"> | 
-|[Emre Girgin](https://github.com/bounswe/bounswe2020group4/wiki/Emre-Girgin)|[Berke Özdemir (Former)](https://github.com/bounswe/bounswe2020group4/wiki/Berke-%C3%96zdemir)|[Katariina Korolainen(Former)](https://github.com/bounswe/bounswe2020group4/wiki/Katariina-Korolainen)|
-| <img src="https://avatars.githubusercontent.com/berkayalkan" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/cagric0" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/olcaytoturker" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/alprnbg" alt="alt text" width="150"> |
-|[Berkay Alkan](https://github.com/bounswe/bounswe2020group4/wiki/Berkay-Alkan)|[Çağrı Çiftçi](https://github.com/bounswe/bounswe2020group4/wiki/%C3%87a%C4%9Fr%C4%B1-%C3%87ift%C3%A7i)|[M. Olcayto Türker](https://github.com/olcaytoturker)|[Alperen Bağ](https://github.com/bounswe/bounswe2020group4/wiki/Alperen-Bağ)|
-| <img src="https://avatars.githubusercontent.com/multiminparvo" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/BCGpp" alt="alt text" width="150"> | <img src="https://avatars.githubusercontent.com/merijjeyn" alt="alt text" width="150"> ||
-|[Veli Can Ünal](https://github.com/bounswe/bounswe2020group4/wiki/Veli-Can-%C3%9Cnal)|[Berke Can Gürer](https://github.com/bounswe/bounswe2020group4/wiki/Berke-Can-G%C3%BCrer)|[Meriç Üngör](https://github.com/bounswe/bounswe2020group4/wiki/Meri%C3%A7-%C3%9Cng%C3%B6r)|| 
+{"attributes":[{"name":"size","value":"45"},{"name":"color","value":"Black"}],"quantity":45}
```

Comment:
Did you commit that file accidentally? :)

Reply:
Yes unfortunately.